jerry-debugger/jerry-client-ws.py

```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default="",
+                        help="set display range")
+    parser.add_argument("--exception", action="store", default="",
```

Comment:
As I see the `--exception` can only have two value: `0` and `1`. Could you please update that only there values are allowed (there is a choice action) also the default should be None and we can specify the type to be int. If the choices are present then we would not need for the try-except below and the help can be extended to contain the meaning of the values.